main.tf

```
@@ -0,0 +1,40 @@
+provider "aws" {
+    region = "ap-southeast-1"
+}
+
+module "module-network" {
+    source = "/home/dang/modules/network"
```

em có sửa đường dẫn theo anh rồi mà nó k nhận. Nên em để như này hic

---

Không được sử dụng dường dẫn tuyệt đối, vì nó chỉ đúng ở trên máy của em 
```suggestion
    source = "network"
```